### PR TITLE
Cephfs fragments

### PR DESCRIPTION
**Pull request by shuchitajnn**
# Description

Added tier-1 fs fragmens

### CODE REVIEW

**Comment by psathyan** [comment]
Would you be able to squash commit numbers 3 to 5. Commit #1 and #2 make sense...

File: suites/pacific/cephfs/tier-1/test-snapshot/07_test_deploy_cluster_with_shell_commands/override.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      args: "ceph osd pool create cephfs-data-ec 64 erasure"
+  - test:
+      index: 3
+      args: "ceph osd pool create cephfs-metadata 64"
+  - test:
+      index: 4
+      args: "ceph osd pool set cephfs-data-ec allow_ec_overwrites true"
+  - test:
+      index: 5
+      args: "cepj fs new cephfs-ec cephfs-metadata cephfs-data-ec --force"
```
Comment (psathyan) [inline suites/pacific/cephfs/tier-1/test-snapshot/07_test_deploy_cluster_with_shell_commands/override.yaml:16]: I believe it should `ceph`

File: fragments/rgw/tier0_pacific_rgw/01_setup_prerequisites.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+../../setup/test_setup_configure_system.yaml
```
Comment (psathyan) [inline fragments/rgw/tier0_pacific_rgw/01_setup_prerequisites.yaml:1]: This being a suite, it should move to `suites/pacific/rgw/tier-0_test-build-sanity`

**Review by psathyan — COMMENTED** [review]
Thanks.

The folder structure for ceph FS looks good.

File: suites/pacific/cephfs/tier-1/test-snapshot/07_test_deploy_cluster_with_shell_commands/override.yaml (hunk 1 above)
Comment (shuchitajnn) [reply suites/pacific/cephfs/tier-1/test-snapshot/07_test_deploy_cluster_with_shell_commands/override.yaml:16]: Right, sorry I will correct it.

File: fragments/rgw/tier0_pacific_rgw/01_setup_prerequisites.yaml (hunk 2 above)
Comment (shuchitajnn) [reply fragments/rgw/tier0_pacific_rgw/01_setup_prerequisites.yaml:1]: Sure, I will do that.

**Pull request closed without merging**